META.json

```diff
@@ -1,41 +0,0 @@
-{
```

Comment:
Since these are used by pause/metacpan etc I am uncomfortable removing them at this time since I do not know how it will impact things.

Perhaps we can delve into the current state of [MY]META.[yml|json] for the echo system and then can fixup all the modules to do the right-thing-du-jour.

Reply:
Like I said, when you run 'perl Makefile.PL' and then 'make dist', the files appear in the distribution. I've attached output from my terminal session proving this to be the case. The tarball that is generated by this can be uploaded to PAUSE without worry.

`xan-dev ~/Projects/Unix-PID-Tiny$ git status
On branch CPANEL-24261_persistent_pidfiles
Untracked files:
  (use "git add <file>..." to include in what will be committed)

	Makefile.old

nothing added to commit but untracked files present (use "git add" to track)
xan-dev ~/Projects/Unix-PID-Tiny$ ls META.*
/bin/ls: cannot access 'META.*': No such file or directory
xan-dev ~/Projects/Unix-PID-Tiny$ perl Makefile.PL
Checking if your kit is complete...
Warning: the following files are missing in your kit:
	META.json
	META.yml
Please inform the author.
Generating a Unix-style Makefile
Writing Makefile for Unix::PID::Tiny
Writing MYMETA.yml and MYMETA.json
xan-dev ~/Projects/Unix-PID-Tiny$ make dist
rm -rf Unix-PID-Tiny-0.92
"/usr/bin/perl" "-MExtUtils::Manifest=manicopy,maniread" \
	-e "manicopy(maniread(),'Unix-PID-Tiny-0.92', 'best');"
mkdir Unix-PID-Tiny-0.92
mkdir Unix-PID-Tiny-0.92/lib
mkdir Unix-PID-Tiny-0.92/lib/Unix
mkdir Unix-PID-Tiny-0.92/lib/Unix/PID
META.yml not found at -e line 1.
mkdir Unix-PID-Tiny-0.92/t
META.json not found at -e line 1.
Generating META.yml
Generating META.json
tar cvf Unix-PID-Tiny-0.92.tar Unix-PID-Tiny-0.92
Unix-PID-Tiny-0.92/
Unix-PID-Tiny-0.92/MANIFEST
Unix-PID-Tiny-0.92/META.yml
Unix-PID-Tiny-0.92/lib/
Unix-PID-Tiny-0.92/lib/Unix/
Unix-PID-Tiny-0.92/lib/Unix/PID/
Unix-PID-Tiny-0.92/lib/Unix/PID/Tiny.pm
Unix-PID-Tiny-0.92/META.json
Unix-PID-Tiny-0.92/.perltidyrc
Unix-PID-Tiny-0.92/Changes
Unix-PID-Tiny-0.92/README
Unix-PID-Tiny-0.92/Makefile.PL
Unix-PID-Tiny-0.92/t/
Unix-PID-Tiny-0.92/t/pod.t
Unix-PID-Tiny-0.92/t/perltidy.t
Unix-PID-Tiny-0.92/t/01.methods.t
Unix-PID-Tiny-0.92/t/perlcritic.t
Unix-PID-Tiny-0.92/t/pod-coverage.t
Unix-PID-Tiny-0.92/t/00.load.t
rm -rf Unix-PID-Tiny-0.92
gzip -9f Unix-PID-Tiny-0.92.tar
Created Unix-PID-Tiny-0.92.tar.gz
xan-dev ~/Projects/Unix-PID-Tiny$ `

Reply:
That output has a warning that they are missing and to inform the author, we should leave them in. Updated w/ the latest info of course.

Reply:
When was the last time you hand-maintained a META.yml and a META.json? Sorry, but I don't see the value in keeping them in source control. I absolutely agree they belong in the distribution, though.

Reply:
What does it hurt to have in source control? I guess they’ve never been in my way probably due to my work flow.

The other benefits are:

# I don’t have to look in the tarball if I need to look at them.
# I know any tools consuming them won’t break.

It may be safe but I’d rather not take that risk at this point in time or as part of unrelated changes.

Worst case I can just add them back I guess.